runtime/runtime.go

```diff
@@ -212,7 +212,7 @@ type interpreterRuntime struct {
 }
 
 // NewInterpreterRuntime returns a interpreter-based version of the Flow runtime.
-func NewInterpreterRuntime(defaultConfig Config) Runtime {
+func NewInterpreterRuntime(defaultConfig Config) *interpreterRuntime {
```

Comment:
`NewInterpreterRuntime` is an exported function, used by FVM, etc. Wouldn't it be better to also return the exported type `Runtime`?

Reply:
Reverted in b6853918adbb10d769b940f51127d4fec560b194